### PR TITLE
test: add ExtraOptionsGsettingsRow widget test 

### PR DESCRIPTION
**Pull request by shan-shaji**

### CODE REVIEW

File: test/widgets/extraopt_gsettings_test.dart [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+    'ExtraOptionsGsettings widget build test',
+    (WidgetTester tester) async {
+      await tester.pumpScreen(
+        widgetBuilder: () => Scaffold(
+          body: ExtraOptionsGsettingsRow(
+            actionLabel: 'Repeat Keys',
+            actionDescription: 'Key presses repeat when key is held down',
+            value: true,
+            onChanged: (_) => {},
+            onPressed: () => {},
```
Comment (mivoligo) [inline test/widgets/extraopt_gsettings_test.dart:18]: This syntax means that `onChanged` and `onPressed` return an empty `Map`. Not sure if that was your intension :)

File: test/widgets/extraopt_gsettings_test.dart [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+    'when value=null returns SizedBox',
+    (WidgetTester tester) async {
+      await tester.pumpScreen(
+        widgetBuilder: () => Scaffold(
+          body: ExtraOptionsGsettingsRow(
+            actionLabel: 'Repeat Keys',
+            actionDescription: 'Key presses repeat when key is held down',
+            value: null,
+            onChanged: (_) => {},
+            onPressed: () => {},
```
Comment (mivoligo) [inline test/widgets/extraopt_gsettings_test.dart:47]: Same here. I guess it should be replaced by: 
```
onChanged: (_) {},
onPressed: () {},
```

File: test/widgets/extraopt_gsettings_test.dart (hunk 1 above)
Comment (shan-shaji) [reply test/widgets/extraopt_gsettings_test.dart:18]: oops, that was not my intention :(

File: test/widgets/extraopt_gsettings_test.dart (hunk 2 above)
Comment (shan-shaji) [reply test/widgets/extraopt_gsettings_test.dart]: @mivoligo  I have fixed it :)

Comment (mivoligo) [reply test/widgets/extraopt_gsettings_test.dart]: Thanks! Sadly I don't have much experience in writing tests. Let's ask @jpnurmi for a review.

Comment (shan-shaji) [reply test/widgets/extraopt_gsettings_test.dart]: Yeah sure :)

**Comment by Feichtmeier** [comment]
@shan-shaji I moved this widget to 
github.com/ubuntu/yaru_widgets
Could you close this pull request here and open it again on that repo? Sorry for the confusion

**Comment by shan-shaji** [comment]
Yeah, sure

**Pull request closed without merging**